main.py

```diff
@@ -854,7 +854,9 @@ def _uni_from_dict(self, uni_dict):
             tag = TAG.from_dict(tag_dict)
         else:
             tag = None
-        uni = UNI(interface, tag)
+
+        priority = uni_dict.get("sb_priority", None)
+        uni = UNI(interface, priority, tag)
```

Comment:
This line needs a change from Kytos repository PR [#303](https://github.com/kytos-ng/kytos/pull/303)

Reply:
@Alopalao thanks for anticipating this PR, but double checking, we'll need this PR if we'll implement `sb_priority` at UNI level, right? Which we'll delegate to a blueprint to better understand first the use cases https://github.com/kytos-ng/mef_eline/pull/214#discussion_r1018032008? 

Let me know if I've lost part of the context of the discussion.

Reply:
Oh, I talked with @italovalcy about the `sb_priority` inside each UNI. It will affect the its respective UNI and the `sb_priority` outside of them will affect the NNIs and egress flows. But I am gonna ask for his review again to make sure.